incident.go

```diff
@@ -380,4 +380,46 @@ type ListAlertResponse struct {
 	Alerts []Alert `json:"alerts,omitempty"`
 }
 
-/* TODO: Manage Alerts, Get Alert, Create Status Updates, Create Responder Request, */
+// CreateIncidentResponderRequestResponse
+type CreateIncidentResponderRequestResponse struct {
```

Comment:
I think we'll go with a shorter name. Will you remove all the `CreateIncident`s and start the names at `ResponderRequest`? So, this struct would be `ResponderRequestResponse`.